pages/workers/[id]/edit/[[...stepId]].tsx

```diff
@@ -33,6 +33,13 @@ const UpdateWorker = (): React.ReactElement => {
   if (!ATeams || !CTeams || !data) {
     return <Spinner />;
   }
+  ATeams.sort(function (a, b) {
+    return a.name.localeCompare(b.name);
+  });
```

Comment:
This looks good, one suggestion that I have is that the sorting could be done in one line, using an arrow function:
`ATeams.sort((a, b) => a.name.localeCompare(b.name));`